bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx93-es.html

```diff
@@ -0,0 +1,57 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}wnp93-v3{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_93') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  {% set landing_source = '?source=' + campaign if campaign else '' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-lg">
+      <h1 class="wnp-pre-title">Es el mes de la ciberseguridad</h1>
+      <h2 class="wnp-main-title">Es el momento de disfrutar de auténtica privacidad online</h2>
+
+      <div class="wnp-main-image video-container">
+        <video id="video" preload="metadata" autoplay loop poster="{{ static('img/firefox/whatsnew/whatsnew93/hero-v3.svg') }}">
+          <source src="{{ static('img/firefox/whatsnew/whatsnew93/hero-v3.webm') }}" type="video/webm">
+        </video>
+      </div>
+
+      <p class="wnp-main-tagline">Mantén tu ubicación e historial de navegación a salvo de miradas indiscretas con una red privada virtual de la marca tecnológica más fiable.</p>
+
+      <div class="wnp-main-cta">
+        <a href="{{ url('products.vpn.landing') }}{{ landing_source }}" class="mzp-c-button mzp-t-product mzp-t-lg" data-cta-text="Get Mozilla VPN" data-cta-type="button">Consigue Mozilla VPN</a>
```

Comment:
Not a blocker as you already have this coded and working, but there's a [new helper](https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#tracking-same-site-links-for-mozilla-vpn) now so we don't need to pass along the `source` parameter. This is in use on the main site navigation in production now, but we haven't used it in a WNP yet. Probably not owrth changing now, but it's here for next time.

Reply:
Yep you should be able to pass a different ID for each variation